src/SFA.DAS.Commitments.Api/Orchestrators/EmployerOrchestrator.cs

```diff
@@ -215,5 +218,37 @@ public EmployerOrchestrator(IMediator mediator, ICommitmentsLogger logger)
                 UserId = userId
             });
         }
+
+        public async Task<GetPendingApprenticeshipUpdateResponse> GetPendingApprenticeshipUpdate(long accountId, long apprenticeshipId)
+        {
+            _logger.Info($"Getting pending update for apprenticeship {apprenticeshipId} for employer account {accountId}");
```

Comment:
AccountId is logged. ProviderId we don't have (on employer side) at the time we make this request. We could however log it when the response comes back (assuming it was found)?